src/ValidatorRegistry.sol

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.20;
+
+// OpenZeppelin Upgradeable Contracts
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+// Interface
+import "./IValidatorRegistry.sol";
+
+/**
+ * @title ValidatorRegistry
+ * @dev Registry for managing validator contracts.
+ *      Implements UUPSUpgradeable for upgradability.
+ */
+contract ValidatorRegistry is
+    Initializable,
```

Comment:
Initialized is already inherited from UUPSUpgradeable

Reply:
No clue what that means just assumed it needed to be in each contract to set initial parameters